app/assets/stylesheets/admin/views/_editions-index.scss

```diff
@@ -21,7 +21,15 @@
     }
   }
 
-  .label.force_published {
```

Comment:
is this going to break https://github.com/alphagov/whitehall/blob/19cd7d72de32454d532c195f35b027fa1b3ba6ac/app/views/admin/dashboard/_document_table.html.erb#L15 ?